### DISCUSSION TITLE
1mm MNI template

### DISCUSSION THREAD

**Discussion by max-kat** [maartenmennes/ICA-AROMA#68] [Q&A]
Dear ICA-AROMA experts,

Before ICA_AROMA, I used fsl feat and supplied the feat directory as input for aroma. However, I registered the structural T1 to 1mm MNI space. I read through the code of ICA_AROMA_functions.py and realized that ICA_AROMA transforms the found components to 2mm MNI space for classification. Thus, my warp files actually warp the found components to a slightly different space (1mm). I did not find a resampling happening in between performed by AROMA, or am I mistaken?
**Thus my question is can ICA_AROMA be used with the 1mm MNI template warp?**

Or are the differences introduced by this slight change in dimensions so big, that the hyperplane parameters are no longer correct and thus cannot be used for classification? 

Also just to clarify, ICA_AROMA requires smoothing, right? It was validated with 6mm so does this mean it only works properly with 6mm smoothing kernels or are smaller ones also ok? 

Thank you so much for your help! 

Cheers, 
max

**Comment by maartenmennes** [comment]
Dear Max,
the warping if FSL is calculated independent of the final resolution. Using your warp you can also warp to 2mm. That being said, if you want to stay within the feat directory it is better to warp to the 2mm MNI reference. This will save disk space and make running AROMA easier. You can later on then still decide to e.g., transform your results to 1mm MNI space using the same warp field.

yes, AROMA (or better the ICA part of AROMA) prefers to work with smooth data. This will ensure components are not as fragmented as could happen with unsmooth data.
